packages/core/src/__tests__/data/blocks/section.raw.tsx

```diff
@@ -0,0 +1,30 @@
+import { Show, For } from '@jsx-lite/core';
+import { BuilderElement } from '@builder.io/sdk';
+import { BuilderBlock as BuilderBlockComponent } from '@fake';
+
+export interface SectionProps {
+  attributes?: any;
+  builderBlock?: BuilderElement;
+  maxWidth?: number;
+}
+
+export default function SectionComponent({
+  maxWidth,
+  builderBlock,
+  ...props
+}: SectionProps) {
+  return (
+    <section
+      {...props.attributes}
+      style={
+        maxWidth && typeof maxWidth === 'number' ? { maxWidth } : undefined
+      }
+    >
+      <Show when={builderBlock && builderBlock.children}>
+        <For each={builderBlock?.children}>
+          {(block) => <BuilderBlockComponent key={block.id} block={block} />}
+        </For>
+      </Show>
```

Comment:
```suggestion
     {props.children}
```

so this will work in any react-ish frameworks out of the box (react, react native, solid)

then we can handle this for other frameworks. e.g. for vue, svelt you can do 

```ts
if (block.bindings._text?.replace(/\s+/g, '') === 'props.children') {
  return `<slot></slot>`
}
```

e.g. here https://github.com/BuilderIO/jsx-lite/blob/main/packages/core/src/generators/svelte.ts#L44:L44

for angular, do the same but `return '<ng-content></ng-content'`

I would also add one more `return '<slot></slot>'` implementation for custom elements too (in html.ts)

the other frameworks (e.g. liquid, swift) don't worry about supporting this feature for now

Reply:
@steve8708 there's now some WIP code in this PR that focus on `{children}` codegen